whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/WhoisRdapElasticServiceTestIntegration.java

```diff
@@ -695,19 +699,6 @@ private void assertCommon(RdapObject object) {
         assertThat(object.getRdapConformance(), containsInAnyOrder("rdap_level_0", "cidr0", "nro_rdap_profile_0"));
     }
 
-    private void assertCopyrightLink(final List<Link> links, final String value) {
-        assertThat(links, hasSize(2));
-        Collections.sort(links);
-
-        assertThat(links.get(0).getRel(), is("copyright"));
-        assertThat(links.get(0).getHref(), is("http://www.ripe.net/data-tools/support/documentation/terms"));
-        assertThat(links.get(0).getHref(), is("http://www.ripe.net/data-tools/support/documentation/terms"));
-
-        assertThat(links.get(1).getRel(), is("self"));
```

Comment:
Why were there two elements in `links` previously and only one now?

Reply:
we are removing self, it is specified in jira ticket